subiquity/ui/views/identity.py

```diff
@@ -107,8 +107,8 @@ def validate_hostname(self):
                 "Server name too long, must be less than {limit}"
                 ).format(limit=HOSTNAME_MAXLEN)
 
-        if not re.match(r'[a-z_][a-z0-9_-]*', self.hostname.value):
-            return _("Hostname must match NAME_REGEX, i.e. [a-z_][a-z0-9_-]*")
+        if not re.match(r'[a-z0-9_][a-z0-9_-]*', self.hostname.value):
```

Comment:
With the duplication between these two lines, maybe we could fix the lint and the redundancy by pulling the regex out to a variable when we can then use for both lines.  What do you think?

Reply:
I agree, thanks. I had initially thought to keep the changes as small as possible but I do agree.

I've now checked out the code locally (rather than direct editing on github UI!) and am also testing using 'make dryrun' now. I'm making the change for username as well as hostname, since these are both structured in the same way - I'll update shortly.